redash/query_runner/databend.py

```diff
@@ -61,12 +61,12 @@ def _define_column_type(column_type):
             return TYPE_STRING
 
     def run_query(self, query, user):
-        host = (self.configuration.get("host") or "localhost"),
-        port = (self.configuration.get("port") or 8000),
-        username = (self.configuration.get("username") or None),
-        password = (self.configuration.get("password") or None),
-        database = (self.configuration.get("database") or None),
-        secure = (self.configuration.get("secure") or False),
+        host = self.configuration.get("host") or "localhost"
```

Comment:
`.get` method allows to set the default value:

`self.configuration.get("host", "localhost")`

Reply:
Good point... I've been stuck in Typescript for a while, so much so that I overlooked this one 🤦‍♂️

Reply:
I merged this PR anyway as it's still better than the current state, but @hantmac you're welcome to submit an update to the config fetching code.

Reply:
Aha.. thanks very much. I would like to update to the config fetching code.